### DISCUSSION TITLE
Convert CMYK to multichannel format psd

### DISCUSSION THREAD

**Discussion by Cosnet** [ImageMagick/ImageMagick#3507] [Help]
In Photoshop you can open a cmyk image and then go to mode and change from CMYK to Multi-channel
![image](https://user-images.githubusercontent.com/406094/113602576-c72a1880-9610-11eb-8be2-c61fe3879379.png)
This is for a piece of software that only accepts multi-channel. I am not interested in alpha channels or Spot colors just changing mode to multichannel cmyk. Actually it would be nice to convert from RGB to cmyk to multi-channel cmyk. 
Kind of a newbie with ImageMagick so any explanation and help would be appreciated.

**Comment by snibgo** [comment]
For us to experiment, it would help if you posted a link to a zip file with a small example of a multichannel cmyk and the same image as an ordinary cmyk.

**Reply by Cosnet** [reply]
Hi Thanks for the quick response. The files I work with are big.
Here is a link to the file: https://cgs-oris.d.pr/7wY0Gv
Inside is a cmyk file and the same converted in photoshop to multichannel cmyk.
Thanks for your help!

**Comment by fmw42** [comment]
I do not believe that ImageMagick can do that, because the it converts the CMYK channels to spot colors. ImageMagick does not support spot colors currently as far as I know. But the IM developers can comment further.

Here is some documentation on Multi-channel mode:


```
Multichannel mode

Multichannel mode images contain 256 levels of gray in each channel and are useful for specialized printing. Multichannel mode images can be saved in Photoshop, Large Document Format (PSB), Photoshop 2.0, Photoshop Raw, or Photoshop DCS 2.0 formats.

These guidelines apply when converting images to Multichannel mode:

Layers are unsupported and therefore flattened.

Color channels in the original image become spot color channels in the converted image.

Converting a CMYK image to Multichannel mode creates cyan, magenta, yellow, and black spot channels.

Converting an RGB image to Multichannel mode creates cyan, magenta, and yellow spot channels.

Deleting a channel from an RGB, CMYK, or Lab image automatically converts the image to Multichannel mode, flattening layers.

To export a multichannel image, save it in Photoshop DCS 2.0 format.
```
